girder/api/v1/system.py

```diff
@@ -121,6 +122,21 @@ def getSetting(self, key, list):
             self.requireParams({'key': key})
             return Setting().get(key)
 
+    @access.public()
+    @autoDescribeRoute(
+        Description('Get publicly accessible settings.')
+    )
+    def getPublicSettings(self):
+        publicSettings = [
+            SettingKey.BRAND_NAME,
+            SettingKey.CONTACT_EMAIL_ADDRESS,
+            SettingKey.PRIVACY_NOTICE,
+            SettingKey.BANNER_COLOR,
+            SettingKey.REGISTRATION_POLICY,
+            SettingKey.ENABLE_PASSWORD_LOGIN,
+        ]
+        return {k: Setting().get(k) for k in publicSettings}
```

Comment:
It would be good to add a batch lookup function in the setting model to avoid this many RTTs to mongo.

Reply:
Agree. Also the current admin settings page queries a much longer list of settings from the `getSetting` method above which has a similar structure to this.